tests/wpt/mozilla/tests/mozilla/bluetooth/idl-Bluetooth.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+'use strict';
+test(() => {
+    //assert_throws('ReferenceError', function () { new Bluetooth() });
+    //assert_throws(null, function () { Bluetooth() });
+    assert_true('requestDevice' in window.navigator.bluetooth);
+}, 'Bluetooth IDL test');
```

Comment:
I don't think so. We ported this test from chrome, and they don't have idlharness test for WebBluetooth, maybe due to the frequently API and WebIDL changes. Moreover our webidl files differ in some places from the spec WebIDL files, and we plan to resolve this in the near future. After these changes landed I think we can create the idlharness test for WebBluetooth. Until then, can we keep these idl test files or should we remove them?